src/layouts/govuk.njk

```diff
@@ -26,8 +26,13 @@
 {% block content %}
   <div class="govuk-o-width-container">
     <main class="govuk-o-main-wrapper">
+    <h1 class="govuk-heading-l">
+      {% block title %}
+      {% endblock %}
+    </h1>
       {% block main %}
         No content rendered to 'main' block
+        }
```

Comment:
Is this meant to be here. It looks unmatched to me.

Reply:
No, good spot.